lib/source_file.h

```diff
@@ -305,8 +306,8 @@ class InputSources final {
        string describing a position in the sources, e.g.:
        int<32> variable;
                ^^^^^^^^ */
-    cstring getSourceFragment(const SourcePosition &position, bool useMarker) const;
-    cstring getSourceFragment(const SourceInfo &position, bool useMarker) const;
+    cstring getSourceFragment(const SourcePosition &position, int trimWidth, bool useMarker) const;
+    cstring getSourceFragment(const SourceInfo &position, int trimWidth, bool useMarker) const;
```

Comment:
Please add a breaf mention of the `trimWidth`'s meaning to the comment, including the fact it has to be at least 10 to work.